l10n_us_partner_name/__manifest__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
# Copyright 2016 LasLabs Inc.
# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).

{
    'name': 'American Partner Name',
    'version': '10.0.1.0.0',
    'author': "LasLabs, Odoo Community Association (OCA)",
    'category': 'Contact Management',
    "website": "https://laslabs.com",
    "license": "LGPL-3",
    "application": False,
    'installable': True,
    'depends': [
        'partner_firstname',
    ],
}

```

Comment:
Not sure what's up with Github on this display - the file is fine & there is actually data below where it's saying the end is